digits/templates/models/images/classification/new.html

```diff
@@ -376,14 +382,16 @@ <h4 style="display:inline-block;">Python Layers</h4>
               )
         .done(function(data) {
                 data = $.parseJSON(data);
-                $('textarea[name=custom_network]').val(data['network']);
+                editor.setValue(data['network']);
+                editor.setValue(editor.getValue(), 1); // Moves cursor to end of editor
```

Comment:
Sorry Luke, I insist we keep it at the bottom. Do you have a particular reason for wanting to see the start first?

No, seriously; I couldn't care less. I did it because by default it selects _all the text_ by default when set text is Set(). Therefore i just put the cursor at the end: for torch this is often the most convenient as the top is often full of inclusions and stuff I rarely touch.